modules/tasks/tasks/src/main/java/org/motechproject/tasks/domain/Channel.java

```diff
@@ -232,6 +214,14 @@ public void setDisplayName(String displayName) {
         this.displayName = displayName;
     }
 
+    public boolean isProvidesTriggers() {
```

Comment:
Is this meant to be persistable? It seems like it shouldn't but there's a public getter/setter that adds it to the schema. Did you forget the Ignore annotation or am I missing something?